acos_client/client.py

```diff
@@ -93,16 +93,20 @@
 class Client(object):
 
     def __init__(self, host, version, username, password, port=None,
-                 protocol="https", timeout=None, retry_errno_list=None):
+                 protocol="https", max_retries=3, timeout=5, retry_errno_list=None):
```

Comment:
Do we have data or documentation that indicate a good number of `max_retires` or `timeout`?  Not that I disagree with these as a starting point, but just wanted to ask.